src/Collage/Sketchy/Helpers.elm

```diff
@@ -0,0 +1,24 @@
+module Collage.Sketchy.Helpers exposing (rotateList, segments)
```

Comment:
I think we could make a "global" `Helpers` module with these functions, they are not specific to `Sketchy`.

Move it to `Collage.Helpers`? Or `Collage.Helpers.List` and import it qualified under `List`? Than `rotateList` could be exported as `rotate`. I don't like to reuse type names in functions when we could make a module around it.

Reply:
Yep, sounds good. I added `Helpers.List` and moved a few functions in there.